rest_api/sawtooth_rest_api/rest_api.py

```diff
@@ -157,11 +175,19 @@ def main():
             log_configuration(log_dir=log_dir, name="sawtooth_rest_api")
         init_console_logging(verbose_level=opts.verbose)
 
+        try:
+            host, port = rest_api_config.bind[0].split(":")
+            port = int(port)
+        except ValueError as e:
+            LOGGER.error("Error parsing host and port from %s: %s",
```

Comment:
This should be a `print`.  Also, the message should say what the format should be:
`"Unable to parse binding %s: Must be '<host>:<port>'"